.github/workflows/go-tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           registry: false
 
       - name: Set up Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v2.0
         with:
           version: v3.7.1
```

Comment:
setup helm 3.8.0

Reply:
removed it completely - it was used to create repositories file